packages/core/core/test/test-utils.js

```diff
@@ -18,6 +18,7 @@ cache.ensure();
 export const DEFAULT_OPTIONS: ParcelOptions = {
   cacheDir: path.join(__dirname, '.parcel-cache'),
   watchDir: __dirname,
+  baseWatcherOptions: {},
```

Comment:
Should we just call it `watcherOptions`? Why the base?

Reply:
I wanted to make it clear that these are not the only options that exist that can be set for the watcher, e.g. the `watchDir` option above it